tiledb/tests/test_libtiledb.py

```diff
@@ -1805,7 +1805,7 @@ def test_varlen_sparse_all_empty_strings(self):
         # we currently have to write a placeholder at the end to
         # avoid zero-length cell error
         # TODO: follow-up with improved testing for empty var-length/strings
-        A = np.array(["", "", "", "", "", "\x00"], dtype=object)
```

Comment:
We can probably just remove this test in favor of the new one below, but please at least remove the comments here since they are now out of data (this was fixed by Luc a few months ago I think).